src/components/NodeLink/functionUpdateVis.js

```diff
@@ -298,11 +300,22 @@ function updateVis() {
   link = linkEnter.merge(link);
 
   link.classed("muted", false);
-
   link
     .select("path")
-    .style("stroke-width", 3)
-    .style("stroke", edgeColor)
+    .style("stroke-width", d => {
+      if (linkWidthVariable) {
+        return d[linkWidthVariable]
+      } else{
+        return 3
+      }
+    })
```

Comment:
Rather than using an existing variable directly as the link width, it's more likely that we'll want to map a numeric variable to a width using a [d3 continuous scale](https://github.com/d3/d3-scale#continuous-scales). But this is a good start for the infrastructure to handle that.

Reply:
Agreed, that's definitely going to be an issue. I just didn't even think of it since I was only using the group var. I think that we should fix this before merging

Reply:
Fixed in aff4fb6